src/code/entities/bank36.asm

```diff
@@ -1701,28 +1702,44 @@ func_036_49F6::
     ret                                           ; $49FB: $C9
 
 Data_036_49FC::
```

Comment:
Could you label the DiablogTables as well? Or if you dont know the exact name then: DialogTable280, DialogTable281, ...
Local refs are missing.